lib/interface/cli/commands/tokens/create.cmd.js

```diff
@@ -1,44 +1,57 @@
-const debug = require('debug')('codefresh:cli:create:token');
+const _ = require('lodash');
 const Command = require('../../Command');
 const createCmd = require('../root/create.cmd');
-const {
-    generateToken,
-} = require('../../../../logic/api/token');
+const { token } = require('../../../../logic').api;
+
 
 const command = new Command({
-    command: 'tokens',
-    aliases: ['token'],
+    command: 'token [name]',
     parent: createCmd,
-    description: 'Create Codefresh token',
-    usage: 'Create Codefresh token',
+    description: 'Generate Codefresh token',
+    usage: 'Create default token or provide --type and --subject. See options for more details.',
     webDocs: {
         category: 'Tokens',
-        title: 'Tokens tokens',
+        title: 'Create tokens',
+        weight: 10,
     },
     builder: (yargs) => {
         return yargs
             .option('subject', {
-                describe: 'name of the token subject',
-                required: true,
+                describe: 'Name of the token subject',
             })
             .option('type', {
                 describe: 'Type of the subject',
-                default: 'runtime-environment',
-                options: ['runtime-environment'],
+                choices: ['runtime-environment', 'user'],
+                default: 'user',
             })
-            .option('name', {
+            .positional('name', {
                 describe: 'token name',
                 required: true,
             })
-            .example('codefresh create token --subject my-k8s-cluster/namespace --name new-token', 'Create token form runtime environment');
+            .example('codefresh create token some-token', 'Create token with default type')
+            .example(
+                'codefresh create token some-token --subject my-k8s-cluster/namespace --type runtime-environment',
+                'Create token from runtime environment',
+            );
     },
     handler: async (argv) => {
-        const res = await generateToken({
+        if (_.isEmpty(argv.name)) {
+            console.log('Token name must be provided');
+            return;
```

Comment:
i think we need to crash here: `process.exit(1)`
check what is the behavior else where